crates/eframe/Cargo.toml

```diff
@@ -92,7 +92,15 @@ web_screen_reader = [
 ]
 
 ## Use [`wgpu`](https://docs.rs/wgpu) for painting (via [`egui-wgpu`](https://github.com/emilk/egui/tree/master/crates/egui-wgpu)).
+##
 ## This overrides the `glow` feature.
+##
+## If you're targetting the web, you also need to turn on the
+## `webgl` and/or `webgpu` features of the `wgpu` crate.
```

Comment:
Maybe worth noting that `webgpu` is enabled by default on wgpu.
I just called them out explicitly in the demo's Cargo.toml for visibility.